src/cb_admin/pages/ConfirmPassword.js

```diff
@@ -25,24 +26,33 @@ export default class ConfirmPassword extends React.Component {
 
     this.state = {
       errors: {},
+      email: null,
     };
   }
 
+  componentDidMount() {
+    CbAdmin.get()
+      .then(res => this.setState({ email: ResponseUtils.getResponse(res).email }))
```

Comment:
This can fail, since `getResponse` may return `null`. Maybe what you want is something closer to `pathOr`:
```js
getResponse('email', res); // returns res.data.result.email, or null if it doesn't exist
```

Reply:
😳 But how did the cb_admin login if they don't have an email?

Reply:
Best not to build assumptions about use cases into functions, especially if they're generic/utility functions.